cli/web_worker_behavior.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
If you're going to call this web_worker_behavior.rs then rename workers.rs to web_worker.rs

Reply:
`web_workers_behavior` might be a bad name, but this is specific to user-space workers.